llama_index/callbacks/base.py

```diff
@@ -76,8 +76,8 @@ def __init__(self, handlers: List[BaseCallbackHandler]):
         """Initialize the manager with a list of handlers."""
         self.handlers = handlers
         self._trace_map: Dict[str, List[str]] = defaultdict(list)
-        self._trace_stack: List[str] = [BASE_TRACE_ID]
-        self._trace_id: Optional[str] = None
+        self._trace_event_stack: List[str] = [BASE_TRACE_EVENT]
+        self._trace_id_stack: Optional[List[str]] = None

```

Comment:
is there a conceptual reason why _trace_id_stack would be optional type on `__init__` but `_trace_event_stack` is a list with one item on `__init__`? find it a tad confusing

Reply:
Early morning code I guess lol let me double check if it's fine to make it non optional type